tests/Sinch.Tests/e2e/Fax/ServicesTests.cs

```diff
@@ -0,0 +1,162 @@
+            WebhookContentType = CallbackUrlContentType.ApplicationJson,
+            DefaultForProject = true,
+            DefaultFrom = "+12015555555",
+            NumberOfRetries = 5,
+            RetryDelaySeconds = 120,
+            ImageConversionMethod = ImageConversionMethod.Halftone,
+            SaveOutboundFaxDocuments = true,
+            SaveInboundFaxDocuments = true
+        };
+
+        [Fact]
+        public async Task Create()
+        {
+            var response = await FaxClient.Services.Create(new CreateServiceRequest()
+            {
+                Name = "hello-service",
+                IncomingWebhookUrl = "http://service.webhook/handler",
+                WebhookContentType = CallbackUrlContentType.ApplicationJson,
+                DefaultForProject = true,
+                DefaultFrom = "+12015555555",
+                NumberOfRetries = 5,
+                RetryDelaySeconds = 120,
+                ImageConversionMethod = ImageConversionMethod.Halftone,
+                SaveOutboundFaxDocuments = true,
+                SaveInboundFaxDocuments = true
+            });
+            response.Should().BeEquivalentTo(_service);
+        }
+
+        [Fact]
+        public async Task Get()
+        {
+            var response = await FaxClient.Services.Get(_service.Id!);
+            response.Should().BeEquivalentTo(_service);
+        }
+
+        [Fact]
+        public async Task Update()
+        {
+            var response = await FaxClient.Services.Update(new UpdateServiceRequest()
+            {
+                Id = _service.Id!,
+                Name = "hello-service",
+                IncomingWebhookUrl = "http://service.webhook/handler",
+                WebhookContentType = CallbackUrlContentType.ApplicationJson,
+                DefaultForProject = true,
+                DefaultFrom = "+12015555555",
+                NumberOfRetries = 5,
+                RetryDelaySeconds = 120,
+                ImageConversionMethod = ImageConversionMethod.Halftone,
+                SaveOutboundFaxDocuments = true,
+                SaveInboundFaxDocuments = true
+            });
+            response.Should().BeEquivalentTo(_service);
+        }
+
+        [Fact]
+        public async Task Delete()
+        {
+            var op = () => FaxClient.Services.Delete(_service.Id!);
+            await op.Should().NotThrowAsync();
+        }
+
+        [Fact]
+        public async Task List()
+        {
+            var response = await FaxClient.Services.List(1, 2);
+            response.Should().BeEquivalentTo(new ListServicesResponse()
+            {
+                Services = new List<Service>()
+                {
+                    _service, _service
+                },
+                PageNumber = 1,
+                PageSize = 2,
+                TotalItems = 2,
+                TotalPages = 1,
+            });
+        }
+
+        [Fact]
+        public async Task ListForNumber()
```

Comment:
As mentioned in another comment, this test can be renamed `ListEmailsForNumber` to make it explicit about which object is queried.